README.md

```diff
@@ -168,6 +168,9 @@ $ python setup.py install
 
 You can find RocketPy's documentation at [Read the Docs](https://rocketpyalpha.readthedocs.io/en/latest/).
 
+## Class diagram
+![Diagram](docs/static/Fluxogram-Page-2.svg)
+
```

Comment:
I like what you have done very much! Creating a new section for the Class diagram is a good option. Did you also consider adding the diagram after line 183? There, RocketPy's main classes are mentioned for the first time, perhaps helping to contextualizing the whole thing.